src/main/java/com/comcast/zucchini/AbstractZucchiniTest.java

```diff
@@ -264,7 +264,13 @@ else if(result.isJsonObject()) {
                 }
             }
 
-            cleanup(context);
+            try {
+                cleanup(context);
+            }
+            catch(Throwable t) {
+                LOGGER.error("ERROR: {}", t);
```

Comment:
We're doing a similar catch for setup() and setupFormatter().  Do you think it would be better if I changed it to catch a RuntimeException? 

As a side effect of the runWith() method throwing, a failed test will actually not be registered as a  failed test, so tests that throw in a parallel run can actually cause a failing set of tests to still register as passing. 

One way that I could fix this is by reversing the logic for failed tests so that it only registers passing tests, but this current logic would also halt serial runs and prevent other contexts from running, as it would terminate the run() method.